home/modules/contribute/pages/playbook.adoc

```diff
@@ -31,18 +31,74 @@ You've already completed the steps on this page.
 . Change into the repository folder created by the clone operation.
 
  $ cd docs-site
-
-== Sync with the Upstream Repository
-
-You need to connect your local repository to the upstream repository so you can use the latest playbooks and extensions.
-
-. Go to the {url-git-playbook}[upstream Docs Site repository^] and click btn:[Clone or download].
-. Click the Clipboard icon to copy the repository's Git URL.
-. In your terminal type `git remote add upstream`, paste the copied URL, and press kbd:[Enter].
-
-Now you're ready to run Antora and build the documentation site on your device.
+ 
+== Configuring the Playbook
+
+To preview your changes on the site locally, you need to configure the playbook to use your local repository.
+
+. Copy the Antora playbook, _antora-playbook.yml_, and save it in the _docs-site_ repository.
+. Rename it _local-antora-playbook.yml_.
+. Open _local-antora-playbook.yml_ in Atom or your editor.
+. Under `sources`, find the `url` key that matches the upstream repository you plan to submit your changes to.
+Replace the repository URL with either the relative system path from the playbook project to your local repository or the absolute path.
+For example, if you're making changes to files in the _docs-server_ repository, you'd enter:
++
+[source,yaml]
+----
+- url: ./docs-server
+----
+
+. If there isn't a `branches` key on the line directly below the `url` key, add it and assign it the list of version branch names, except for the version branch your working branch is based on.
+In place of the remote version branch, use the value of `HEAD`.
+`HEAD` tells Antora to use your working branch.
+For example, if your working branch is based off of the docs-server release/5.5 branch, you'd assign the following values to `branches`:
++
+[source,yaml]
+----
+- url: ./docs-server
+  branches: [release/4.*, release/5.0, release/5.1, HEAD]
```

Comment:
If we put 7.0, 71., 6.*
it means ppl can c+p this if they prefer

Reply:
Yep, good idea.